sdk/include/reactos/drivers/afd/shared.h

```diff
@@ -110,6 +110,8 @@ typedef struct _AFD_SEND_INFO_UDP {
     ULONG				AfdFlags;
     TDI_REQUEST_SEND_DATAGRAM		TdiRequest;
     TDI_CONNECTION_INFORMATION		TdiConnection;
+    PVOID				Address; // for CORE-17521
+    PINT				AddressLength; // for CORE-17521
```

Comment:
This variable doesn't appear to be used in this PR.

Reply:
removed using web editor